code/datums/supplypacks/robotics_vr.dm

```diff
@@ -0,0 +1,34 @@
+/datum/supply_packs/robotics/mecha_gopher
```

Comment:
Do we even have these for normal mechs? If not, we should not have them here either. I am very certain these circuit crates were removed.

Reply:
![image](https://user-images.githubusercontent.com/16435949/39663876-93530c6c-502f-11e8-923b-38a8a72cd688.png)
the non combat mechs have their boards here so should i remove the two combat mech crates?

Reply:
Nope, guess we do have those after all.

Reply:
Disregard my request to remove things.

Reply:
well it would be reasonable to request the removal of the combat mech boards, would you like that?